setup.py

```diff
@@ -47,23 +52,30 @@
     ],
     package_dir={"": "src"},
     packages=setuptools.find_packages(where="src"),
-    python_requires=">=3.6",
+    include_package_data=True, 
+    package_data={
+        "": ["*.sh", "*.crt", "*.key"], 
+    },
+    python_requires=">=3.8",
```

Comment:
The requirement is changaed from 3.6 to 3.8. Any reasons for higher versions?

Reply:
Globus Compute requires a higher version of python